app/scripts/modules/core/src/application/config/trafficGuard/trafficGuardConfig.component.ts

```diff
@@ -37,7 +37,9 @@ export class TrafficGuardConfigController {
     isDirty: false,
   };
 
-  public constructor(private accountService: AccountService, private namingService: NamingService) { 'ngInject'; }
+  public constructor(private $log: ng.ILogService,
```

Comment:
I had that originally but compilation fails with: `Cannot find name 'ILogService'`